cmd/release-controller/http.go

```diff
@@ -647,6 +647,15 @@ func (c *Controller) httpReleaseInfo(w http.ResponseWriter, req *http.Request) {
 				out = fmt.Sprintf("## Changes from %s\n%s", info.Previous.Name, changed)
 			}
 			out = rePrevious.ReplaceAllString(out, fmt.Sprintf("$1[%s](/releasetag/%s)$2", info.Previous.Name, info.Previous.Name))
+
+			// add link to tag from which current version promoted from
+			rePromoted, err := regexp.Compile(fmt.Sprintf("Promoted from (.*):(.*)"))
```

Comment:
Move the regex compilation to a constant `rePromotedFrom` outside this function (like `rePrevious`) as `MustCompile()`